jdaviz/core/custom_units.py

```diff
@@ -6,6 +6,13 @@
 PIX2 = u.pix * u.pix
 
 
+# Add spaxel to enabled units
+def enable_spaxel_unit():
+    spaxel = spaxel = u.Unit('spaxel', represents=u.pixel, parse_strict='silent')
```

Comment:
This is already in specutils parser (but not exposed). Should this be moved upstream eventually?

Reply:
I think so, for now I just made our spaxel definition match the specutils one so they would actually divide properly. And whoops, looks like I have a double `spaxel = ` there.